devdoc/Releasing_Tycho.md

```diff
@@ -22,44 +47,19 @@ This page describes the steps necessary to create releases of the Tycho project.
 </settings>
 </source>
 ```
-- [ ] Checkout the tag version `tycho-<VERSION>` so current commit is the tag.
+
+- [ ] Checkout branch `tycho-N.M.x`
+- [ ] Update version to remove `-SNAPSHOT` with `mvn org.eclipse.tycho:tycho-versions-plugin:set-version -DnewVersion=<VERSION>`
+- [ ] Update versions in tycho-demo folder
```

Comment:
could this be done automatic or does this need manual adjustments?

Reply:
It could probably be done automatically, but that's not in the scope of this change.